nvm.sh

```diff
@@ -2317,6 +2389,49 @@ nvm_npmrc_bad_news_bears() {
   return 1
 }
 
+nvm_windows_drive_posix() {
+  nvm_grep -iEe "^$1 " /proc/mounts 2>/dev/null | command awk '{ print $2 }'
+}
```

Comment:
As this is Windows related I dont think that it can be added among the existing unit tests that ran under Linux but I have already added some tests in windows-npm.yml